IfIsNotInA.java

```diff
@@ -0,0 +1,35 @@
+
+
+
+
+            /*21.	Given two array lists A and B, A has an element that is not in B while the other
+             elements are the same in both two lists. Find that missing element in B.
+              (This is a live coding challenge. Any programming language is fine.)*/
+
+
+import java.util.*;
+
+public class IfIsNotInA {
+    public static List solution(int[] first, int[] second) {
+        ArrayList<Integer> firstArrayList = new ArrayList<Integer>();
+        ArrayList<Integer> secondArrayList = new ArrayList<Integer>();
+        for (int i = 0; i < first.length; i++) {
```

Comment:
Learn to use java8 streams, it helps you do these things with a much more concise syntax, and it's less prone to human error.

https://www.youtube.com/watch?v=Q93JsQ8vcwY
https://www.baeldung.com/java-8-streams